server/templatize.js

```diff
@@ -3,7 +3,13 @@
 let Nunjucks = require("nunjucks");
 
 module.exports = function templatize(server, paths) {
-  let engine = new Nunjucks.Environment(paths.map(path => new Nunjucks.FileSystemLoader(path)), { autoescape: true });
+let nunjucksOptions = {
+	noCache: true,
```

Comment:
Please indent these lines with spaces instead of tabs, I left a comment in the Conversation tab explaining how. Cheers!